### PR TITLE
About soiree

### PR DESCRIPTION
**Pull request by VamsiRao**
Solving issue #3 and #5

### CODE REVIEW

File: public/wp-content/themes/ColoredCow/landing.php [hunk 1]
```diff
@@ -7,7 +7,86 @@
     get_header();
 ?>
 
+<div class="container">
+	<div class="row">
+		<div class="col-xs-12 col-md-6">
+			<h5 class="soiree-header font-marcellus">Soiree</h5>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: use css for font-sizing. in place of h5

File: public/wp-content/themes/ColoredCow/landing.php [hunk 2]
```diff
@@ -7,7 +7,86 @@
...
+			<p class="soiree-text font-inherit" >ColoredCow celebrates every first Saturday of the month with family and friends. This custom has been started to take a little time off from work and enjoy some moments in life. we believe in sharing moments and learning with each other. Come and join us over music, food, drinks and some moments full of laughter and joy.</p>
+			<hr>				
+			<div class="col-md-odffset-1 col-md-12 col-xs-12 text-center">
+				<h5 class="join-header font-marcellus">Wanna Join The Party?</h5>				
+				<button type="button" class="btn btn-outline-warning btn-lg join-button" data-toggle="modal" data-target="#request_modal">Request Invite</button>
+			</div>
+		</div>
+		<div class="col-xs-12 col-md-6 font-marcellus event-col">
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php]: what naming the class as font-marcellus??

File: public/wp-content/themes/ColoredCow/landing.php [hunk 3]
```diff
@@ -7,7 +7,86 @@
...
+						</div>
+						<div class="form-group  col-sm-12">
+							<label for="request_phone" >Phone Number</label>
+							<input type="number" class="form-control" id="request_phone" name="request_phone" placeholder="Your Phone No." required>
+						</div>
+						<div class="col-sm-12">
+							<strong>Gender:</strong>																
+							<div class="row">
+								<div class="col-sm-12">
+								</div>
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/landing.php:64]: whats inside this div?

File: public/wp-content/themes/ColoredCow/src/scss/fonts.scss [hunk 4]
```diff
@@ -7,4 +7,10 @@
 }
 .font-oswald{
 	font-family: Oswald;
+}
+.font-marcellus{
+	font-family: Marcellus SC;
+}
+.font-inherit{
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/fonts.scss]: Let's not use this. Read about inheritance in scss

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 5]
```diff
@@ -67,15 +67,61 @@ Text Domain: coloredcow
...
 
 }
 
+.soiree-header{
+	color:$goldenrod;
+	font-size:85px;
+}
+
+.soiree-text{
+	margin-top:30px;
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss:89]: you can extend the font class here

File: public/wp-content/themes/ColoredCow/src/scss/style.scss [hunk 6]
```diff
@@ -67,15 +67,61 @@ Text Domain: coloredcow
...
+	padding:10px;
+	padding-top:70px;
+	text-align:center;
+}
+
+.event-name{
+	font-size:60px;
+}
+
+.event-date-loc{
```
Comment (Shubham2301) [inline public/wp-content/themes/ColoredCow/src/scss/style.scss]: let's use more explanatory class name

**Review by Shubham2301 — CHANGES_REQUESTED** [review]

File: public/wp-content/themes/ColoredCow/landing.php (hunk 3 above)
Comment (VamsiRao) [reply public/wp-content/themes/ColoredCow/landing.php:64]: for space

**Pull request merged**